includes/tracks/class-wc-tracks.php

```diff
@@ -74,13 +74,9 @@ public static function get_server_details() {
 	 */
 	public static function record_event( $event_name, $properties = array() ) {
 		/**
-		 * Don't track users who haven't opted-in to tracking or if a filter
-		 * has been applied to turn it off.
+		 * Don't track users who don't have tracking enabled.
 		 */
-		if (
-			'yes' !== get_option( 'woocommerce_allow_tracking' ) ||
-			! apply_filters( 'woocommerce_apply_tracking', true )
-		) {
+		if ( ! WC_Site_Tracking::is_tracking_enabled() ) {
```

Comment:
This method uses `woocommerce_apply_user_tracking` filter, so people that is already trying to disable with `woocommerce_apply_tracking` will get tracked... For now we should support both filters, until we can deprecate one.

Reply:
Good catch @claudiosanches. Fixed in https://github.com/woocommerce/woocommerce/pull/23772/commits/c74d5b30caf636e98ac674f250a7eb9c1e5c0c05